scripts/logger.py

```diff
@@ -1,7 +1,6 @@
 import sys
 import os
 from datetime import datetime
-from tabnanny import verbose
```

Comment:
Why this import has been removed?

Reply:
This library was added to make a proof of concept with verbose logs. After some research, we deleted any dependence on this library but we forgot to delete the import.